decidim-admin/app/commands/decidim/admin/create_participatory_space_private_user.rb

```diff
@@ -26,9 +26,10 @@ def call
         return broadcast(:invalid) if form.invalid?
 
         ActiveRecord::Base.transaction do
-          create_or_invite_user
+          @user = @user ||= existing_user || new_user
```

Comment:
This double assign is weird 😕

Reply:
you are doing the same in  `@user = @user ||= existing_user || new_user` in `CreateAssemblyAdmin`... 😕

Reply:
And it's being discussed in the PR that added it. Might be some weird Rubocop change, but it's still weird and non-idiomatic. Can you change it, please? 😄

Reply:
done

Reply:
Sorry - I believe that's a Rubocop automated change 🤷‍♂️

Reply:
So, can this PR be accepted?